distributionviewer/core/static/js/app/router.js

```diff
@@ -6,13 +6,13 @@ import MainLayout from './components/layouts/main-layout';
 
 // Pages
 import Home from './components/home';
-import ExampleChartDetailContainer from './components/containers/example-chart-detail-container';
+import ChartDetailContainer from './components/containers/chart-detail-container';
 
 export default (
   <Router history={browserHistory}>
     <Route component={MainLayout}>
       <Route path="/" component={Home} />
-      <Route path="/chart/:chartId" component={ExampleChartDetailContainer} />
+      <Route path="/chart/:chartName" component={ChartDetailContainer} />
```

Comment:
> Since /metrics/ returns all metrics with only name/descr, does having /metric/<metric_name>/ make sense that it then returns all the data for that metric? If you both like that, it's fine with me.

For the API? So we would hit `/metrics/` for all metrics and `/metrics/architecture` for the architecture points? If so I agree!